standalone/runtime/tests/test_btc_relay.rs

```diff
@@ -10,8 +10,12 @@ type BTCRelayError = btc_relay::Error<Runtime>;
 #[test]
 fn integration_test_submit_block_headers_and_verify_transaction_inclusion() {
     ExtBuilder::build().execute_without_relay_init(|| {
+        // ensure that difficulty check is enabled
+        BTCRelayPallet::set_disable_difficulty_check(false);
+        assert!(!BTCRelayPallet::disable_difficulty_check());
+
         // reduce number of blocks to reduce testing time, but higher than 2016 blocks for difficulty adjustment
-        const BLOCKS_TO_TEST: usize = 2_100;
+        const BLOCKS_TO_TEST: usize = 5_000;
```

Comment:
Incremented this so that we have at least two difficulty checks in the integration tests. Anything else will slow down testing a bit too much.